### PR TITLE
First puppet push

### PR DESCRIPTION
**Pull request by mcgonagle**

### CODE REVIEW

File: manifests/api/install.pp [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+
+class { 'postgresql::server':
+    ip_mask_deny_postgres_user => '0.0.0.0/32',
+    ip_mask_allow_all_users    => '0.0.0.0/0',
+    listen_addresses           => '*',
+    ipv4acls                   => ['local   all             postgres                                trust'],
+    postgres_password          => 'password',
+}
+
+package { "postgresql93-devel":
```
Comment (chriskacerguis) [inline manifests/api/install.pp]: Can you update to use PostgreSQL 9.4.x?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,15 @@
-# puppet-jellyfish
-Jellyfish Puppet Manifests
+jellyfish
+
```
Comment (chriskacerguis) [inline README.md]: Could you please provide some docs on how to run this?

File: Modulefile [hunk 3]
```diff
@@ -0,0 +1,11 @@
+name    'bah-jellyfish'
```
Comment (chriskacerguis) [inline Modulefile]: Shouldn't the name be "puppet-jellyfish"?

File: Modulefile [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+version '0.1.0'
+source 'UNKNOWN'
+author 'bah'
+license 'Apache License, Version 2.0'
+summary 'UNKNOWN'
```
Comment (chriskacerguis) [inline Modulefile]: Can you add a summary and description?

File: manifests/api/install.pp [hunk 5]
```diff
@@ -0,0 +1,200 @@
...
+}
+
+class { 'postgresql::globals':
+    manage_package_repo => true,
+    version             => "${postgres_version}",
+}
+
+class { 'postgresql::server':
+    ip_mask_deny_postgres_user => '0.0.0.0/32',
+    ip_mask_allow_all_users    => '0.0.0.0/0',
```
Comment (chriskacerguis) [inline manifests/api/install.pp]: Is this allowing all users to connect from all IP's?

File: manifests/api/install.pp [hunk 6]
```diff
@@ -0,0 +1,200 @@
...
+    ensure  => latest,
+    require => Class["postgresql::server"]
+}
+
+exec { "bundle":
+    command => "/home/jellyfish/.rbenv/versions/2.2.1/bin/bundle",
+    cwd     => "${jellyfish_home}/api",
+    user    => "${jellyfish_user}",
+    group   => "${jellyfish_group}",
+    path    => "/home/jellyfish/.rbenv/shims:/home/jellyfish/.rbenv/bin:/usr/local/bin:/bin:/usr/bin:/usr/local/sbin:/usr/sbin:/sbin:/home/jellyfish/bin:/usr/pgsql-9.3/bin",
```
Comment (chriskacerguis) [inline manifests/api/install.pp]: Should that path use ${jellyfish_home} and not /home/jellyfish?

**Comment by chriskacerguis** [comment]
@mcgonagle Tom, could you please update this PR with the changes?

File: manifests/api/install.pp (hunk 1 above)
Comment (mcgonagle) [inline manifests/api/install.pp]: Updated to 9.4

File: manifests/api/install.pp (hunk 6 above)
Comment (mcgonagle) [inline manifests/api/install.pp]: Fixed

File: README.md [hunk 7]
```diff
@@ -1,2 +1,94 @@
...
+puppetlabs-concat, '>= 1.2.0'
+pupppetlabs-git, '>= 3.0.0'
+jdowning-rbenv, '1.4.1'
+puppetlabs-stdlib, '4.5.1'
+puppetlabs-vcsrepo, '1.2.0'
+maestrodev-wget, '1.5.7'
+
+
+Usage
+-----
```
Comment (chriskacerguis) [inline README.md]: How does one install Puppet?

**Pull request merged**